File: 260126/배수의 개수/number-of-multipliers.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc=new Scanner(System.in);

        int[] arr=new int[10];
        int[] answer=new int[2];

        for(int i=0; i<10; i++){
            arr[i]=sc.nextInt();
        }

        for(int i=0; i<10; i++){
            if(arr[i]%3==0&&arr[i]%5==0){
                answer[0]+=1;
                answer[1]+=1;
            }
            else if(arr[i]%3==0){
                answer[0]+=1;
            }
            else if(arr[i]%5==0){
                answer[1]+=1;
            }
        }

        for(int i=0; i<2; i++){
            System.out.printf("%d ", answer[i]);
        }
    }
}

File: 260224/배수의 개수/number-of-multipliers.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int cnt_3=0;
    int cnt_5=0;

    for(int i=0; i<10; i++){
        int temp;
        cin>>temp;

        temp%3==0?cnt_3++:false;
        temp%5==0?cnt_5++:false;
    }
    cout<<cnt_3<<" "<<cnt_5;
    return 0;
}